jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -194,21 +197,23 @@ def _create_ejv_file_for_partner(cls, batch_type: str):  # pylint:disable=too-ma
                                                                 line_number, 'D' if not is_reversal else 'C'))
 
                     control_total += 1
-
-            sequence = 1
-            # Create ejv invoice link records and set invoice status
-            for inv in invoices:
-                # Create Ejv file link and flush
-                link_model = EjvInvoiceLinkModel(invoice_id=inv.id,
-                                                 ejv_header_id=ejv_header_model.id,
-                                                 disbursement_status_code=DisbursementStatus.UPLOADED.value,
-                                                 sequence=sequence)
-                # Set distribution status to invoice
-                db.session.add(link_model)
-                sequence += 1
-                inv.disbursement_status_code = DisbursementStatus.UPLOADED.value
-
-            db.session.flush()
+                jv_invoice_order.sort(key=lambda inv: inv.id)
```

Comment:
Why would you sort by invoice id? you already have the list in order as they're inputted remove this